src/app.ts

```diff
@@ -3,31 +3,38 @@
  * Licensed under the MIT License.
  */
 
-import { ISharedMap, SharedMap } from "@fluid-experimental/fluid-framework";
-import type { ContainerSchema } from '@fluid-experimental/fluid-static';
-import TinyliciousClient from '@fluid-experimental/tinylicious-client';
-import { getContainerId } from './utils';
-import { vueRenderView as renderView } from './view';
-
-TinyliciousClient.init();
+import { ContainerSchema, ISharedMap, SharedMap } from "@fluid-experimental/fluid-framework";
+import { FrsClient, FrsConnectionConfig, FrsContainerConfig, InsecureTokenProvider } from "@fluid-experimental/frs-client";
+import { getContainerId } from "./utils";
+import { vueRenderView as renderView } from "./view";
 
 const { id, isNew } = getContainerId();
 
+const config: FrsConnectionConfig = {
```

Comment:
how does this example know if it's running in local mode or not?

Reply:
I pushed some updates to make this clearer. I added a commented out example of a real FRS service config with a comment explaining it.